RationaleMCP/0031/grammar.md

```diff
@@ -265,7 +266,8 @@ end _F;
 ## B26 Equations
 
 > _equation_ →\
-> &emsp; ( _simple-expression_ ( **=** _expression_ )?\
+> &emsp; _decoration_?\
+> &emsp; ( _simple-expression_ _decoration_? ( **=** _expression_ )?\
```

Comment:
Web Meeting (Hans, Henrik, Gerd, Martin, Oliver)
No issue with having no parentheses here, as they are only related to the dropped prefix option.